packages/botonic-plugin-contentful/tests/contentful/contents/text.test.ts

```diff
@@ -1,19 +1,21 @@
 import { expectImgUrlIs } from './image.test'
 import { testContentful, testContext } from '../contentful.helper'
 import * as cms from '../../../src'
+import { ENGLISH } from '../../../src'
```

Comment:
In the future, refactor it because we are import * from 'src', and then another constant from 'src' too. It looks rare!